ui/admin/app/styles/app.scss

```diff
@@ -1083,3 +1083,14 @@
     margin-bottom: 0;
   }
 }
+
+// scope picker
+.scope-picker {
+  &__toggle-button {
+    width: 100%;
+  }
+
+  .indentation {
+    padding-left: 40px;
+  }
```

Comment:
What do you think about doing something like this instead?

```suggestion
  li:has(.indentation),
  li.indentation {
    margin-left: 2rem;
  }
```
The current implementation shows the little blue bar when hovering at the far left which I'm not sure if this was intentional as I would've expected it to be still with the content like in the second screenshot, but maybe this is a design question. I also think we should just use `rems` for this. 

<img width="260" alt="image" src="https://github.com/user-attachments/assets/533ef7bf-80d3-4890-9c27-a0f693334d66" />

<img width="268" alt="image" src="https://github.com/user-attachments/assets/6a40acfb-72bc-43d3-b62f-1fbee25e988f" />

Reply:
great catches!